docs/MODULE.bazel

```diff
@@ -0,0 +1,32 @@
+module(
+    name = "supply-chain-docs",
+    version = "HEAD",
+)
+
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.7.1",
+)
+bazel_dep(
+    name = "stardoc",
+    version = "0.8.0",
```

Comment:
The new CI step took 4.5 minutes. That's annoying for contributors who are just sending a simple PR to fix a docs typo.
https://github.com/bazel-contrib/supply-chain/actions/runs/14680914101/job/41203442699?pr=12

This is happening because stardoc is being built from Java sources, and those take a hard-coded dependency on the protoc `cc_binary` (they seem hesitant to allow me to fix that https://github.com/bazelbuild/stardoc/pull/243)

I recommend using a pre-build of stardoc to avoid that. https://github.com/aspect-build/rules_js/pull/2156/files is what I've done in Aspect rules.

Reply:
I'll leave this for further optimization for now to keep the PR small.